test/e2e/devices/devicegettest/device_get_test.go

```diff
@@ -0,0 +1,74 @@
+package devicestest
+
+import (
+	"io"
+	"os"
+	"strings"
+	"testing"
+
+	root "github.com/equinix/metal-cli/internal/cli"
+	"github.com/equinix/metal-cli/internal/devices"
+	outputPkg "github.com/equinix/metal-cli/internal/outputs"
+	"github.com/equinix/metal-cli/test/helper"
+	"github.com/spf13/cobra"
+)
+
+func TestCli_Devices_get(t *testing.T) {
+	var projectId, deviceId string
+	subCommand := "device"
+	consumerToken := ""
+	apiURL := ""
+	Version := "metal"
+	rootClient := root.NewClient(consumerToken, apiURL, Version)
+	type fields struct {
+		MainCmd  *cobra.Command
+		Outputer outputPkg.Outputer
+	}
+	tests := []struct {
+		name    string
+		fields  fields
+		want    *cobra.Command
+		cmdFunc func(*testing.T, *cobra.Command)
+	}{
+		{
+			name: "get_by_device_id",
+			fields: fields{
+				MainCmd:  devices.NewClient(rootClient, outputPkg.Outputer(&outputPkg.Standard{})).NewCommand(),
+				Outputer: outputPkg.Outputer(&outputPkg.Standard{}),
+			},
+			want: &cobra.Command{},
+			cmdFunc: func(t *testing.T, c *cobra.Command) {
+				root := c.Root()
+				projectId = helper.Create_test_project("get-by-device-id-pro")
```

Comment:
The names for all resources created in e2e tests should start with a common prefix (`cli-test-`, for example) that is unique to this repo so that we can more easily identify and clean up dangling resources when needed.

Reply:
https://go.dev/doc/effective_go#mixed-caps

The helper method name doesn't fit the typical convention.

Reply:
The helper names have been updated, but I mistakenly resolved this conversation about the resource names.  I've copied the comment onto the latest code and will re-resolve this one.